w3af/core/controllers/chrome/login/submit_form/main.py

```diff
@@ -91,3 +93,4 @@ def _handle_exception(self, strategy, e):
                 e,
                 self.debugging_id)
         om.out.debug(msg % args)
+        om.out.error(traceback.format_exc())
```

Comment:
This was just added for debugging?

Reply:
It was added by purpose as I think it's good to have at least traceback saved when exception is silenced. If I wouldn't add this there would be no way to find this traceback as exception is silenced at this point. Showing only the exception name is not enough in many cases. How should this be handled?

Reply:
I'm not sure about the context here, and if it will work, but we have an exception handler that will catch "all exceptions" and save them to a text file in /tmp.

Another alternative, if the scan can continue without a major reduction in functionality and user doesn't need to know about this issue is to also use `debug`?

Reply:
> we have an exception handler that will catch "all exceptions" and save them to a text file in /tmp
Not this time. The exception is not propagated in this place, it's simply silenced.
Changed to `debug` according to your recommendation c3e0eec2b.